codigos_teste/gregori_221004_tamanho-string.py

```diff
@@ -0,0 +1,8 @@
+palavra = input("Digite uma palavra: ")
+
+tamanho = 0
+
+for char in palavra:
```

Comment:
Está correto.
Apenas uma observação. Quando você precisa de uma variável que não terá utilidade alguma, nesse caso, o "char". Pode usar o "_", ex:

for _ in palavra:
 ...

Reply:
show

Reply:
ficou um pouco confuso pq fiz alguma mudanças e alterei a ordem dos exercicios...
mas agora fica faltando só ultimo exercicio.

Reply:
Aqui tu consegue ver as alterações por arquivo. E quando eu coloco algum comentário ele vai aparecer na linha que é... Logo tu se acostuma.